YOLayout/UIKit/YOView.h

```diff
@@ -29,6 +29,9 @@
  */
 @interface YOView : UIView <YOLayoutView> { }
 
+@property YOLayout *viewLayout;
```

Comment:
:+1: I like this better as a name. It's like `collectionViewLayout` except for `UIView` instead of `UICollectionView`.